src/ui/public/doc_table/components/table_row.js

```diff
@@ -77,6 +78,35 @@ module.directive('kbnTableRow', function ($compile) {
 
         $detailsScope.row = $scope.row;
 
+        $detailsScope.copyTextToClipboard = text => {
```

Comment:
Sure! You'll need to do two things:

1. Import the new copy module here: https://github.com/Bargs/kibana/blob/9924a58f6915a0c1a9df0e6b0f713482fdf69f32/src/ui/public/autoload/modules.js#L8-L8 This isn't an angular thing, but it's required so that the module gets executed and added to angular's dependency injection context.

2. Use the service like a normal angular dependency, for example: https://github.com/Bargs/kibana/blob/81efe3b73419acef3c6fc11062511ce4d7e08d60/src/core_plugins/kibana/public/management/sections/indices/add_data_steps/pipeline_setup/directives/output_preview.js#L39-L39